infrastructure/modules/key_vault/main.tf

```diff
@@ -0,0 +1,88 @@
+locals {
+
+}
+
+data "azurerm_client_config" "current" {}
+
+# https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#security
+data "azurerm_role_definition" "key_vault_administrator" {
+  role_definition_id = "00482a5a-887f-4fb3-b363-3b7fe8e74483"
+}
+
+data "azurerm_resource_group" "key_vault" {
+  name = var.resource_group_name
+}
+
+resource "random_string" "key_vault" {
```

Comment:
Veldig ubeskrivende navn paa en random_string.

Reply:
enn så lenge brukes det bare i navnet på keyvault. kan være "key_vault_suffix"

Reply:
Ja - eller `key_vault_name_suffix`. Bare noe mer eksplisit. Og hvis du trenger flere random strings, saa bare lag flere i stedet for aa gjenbruke (med mindre det er gode aarsaker til aa bruke samme navn paa ting).

Reply:
:heavy_check_mark: